var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -21,7 +21,9 @@ class Steps(CMakePackage):
     variant("petsc", default=False, description="Use PETSc library for parallel E-Field solver")
     variant("mpi", default=True, description="Use MPI for parallel solvers")
     variant("coverage", default=False, description="Enable code coverage")
+    variant("bundle", default=True, description="Use bundled libraries")
 
+    depends_on("gdb", when="@develop", type="build")
```

Comment:
gdb? why?

Reply:
In the `spack build-env steps@develop bash`, I want to have gdb whenever I need it. Maybe there is another way to have it easily in the PATH without looking in install_dir manually. any idea?

Reply:
Users also install `develop` just to test latest version. Having gdb as direct dependency is not desirable then. May be separate variant `gdb` ? 

> Maybe there is another way to have it easily in the PATH without looking in install_dir manually. any idea?

Doesn't just `spack load gdb` work if you are in build-env?

Reply:
Thank you @pramodk 
I didn't know about `spack load`
I remove the patch